apps/fishing-map/features/map/map.slice.ts

```diff
@@ -233,23 +237,23 @@ export const fetchFishingActivityInteractionThunk = createAsyncThunk<
         })
       })
 
-      const topHoursVessels = vesselsBySource
+      const topIncidenceVessels = vesselsBySource
```

Comment:
Took me a while to understand what `incidence` meant in that context. It's generic for 'hours or detections' correct? Then why not `topActivityVessels` or simply `topVessels`?